overrides/config/randompatches.cfg

```diff
@@ -110,7 +110,7 @@ client {
 
         # The Minecraft window title.
         # Default: Minecraft 1.12.2
-        S:title=Minecraft 1.12.2
+        S:title=Omnifactory 1.2.2
```

Comment:
1.2.3?

Reply:
We're currently on 1.2.2 + dev commits. I think it would be best to control this value via CICD actually.

Non-tagged commits: `Omnifactory: Dev Snapshot %{commitHash}`
Tagged commits: `Omnifactory v1.2.3`